xarray/plot/plot.py

```diff
@@ -124,14 +124,14 @@ def plot(
     **kwargs,
 ):
     """
-    Default plot of DataArray using matplotlib.pyplot.
+    Default plot of DataArray using :py:mod:`matplotlib:matplotlib.pyplot`.
 
     Calls xarray plotting function based on the dimensions of
-    darray.squeeze()
+    the :py:meth:`squeezed DataArray <xarray.DataArray.squeeze>`.
```

Comment:
```suggestion
    the squeezed DataArray.
```

Reply:
Is it ok to leave this? I think to some it could be helpful to have the link to `.squeeze`.

Reply:
the issue is that rst links are not particularly easy to read (they might still make sense if the advantages outweigh that, but I think we should use them sparingly). Would it help to move the link to the `See Also` section?

Reply:
That makes sense, I will move to See Also.